test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
I don't see Microsoft.AspNet.Mvc package at all in project.lock.json. Furthermore, the only aspnet dependency under .nugget/packages that I see is AspNet.WebApi.Client (used for webapi test application as opposed to the current application).